doclib/msgpack/error.rb

```diff
@@ -1,5 +1,8 @@
 module MessagePack
 
+  class PackError < StandardError
+  end
```

Comment:
This class doesn't have `attr_accessor :error_value`. Is it intentional?

Reply:
Oh that's strange... not intentional, I'll add it back, thanks!